modules/olm-creating-fb-catalog-image.adoc

```diff
@@ -6,7 +6,7 @@ ifdef::openshift-origin[]
 :registry-image: quay.io/operator-framework/opm:latest
 endif::[]
 ifndef::openshift-origin[]
-:registry-image: registry.redhat.io/openshift4/ose-operator-registry:v{product-version}
+:registry-image: registry.redhat.io/openshift4/ose-operator-registry-rhel9:v{product-version}
```

Comment:
I'm assuming "{product-version}" will change to the version number and not Branch Build. Would be worth checking on Pantheon. 

![Screenshot from 2024-07-30 15-25-21](https://github.com/user-attachments/assets/83246c6e-0c1c-417b-9e1a-cb89d7d63855)